src/builder.py

```diff
@@ -111,21 +112,30 @@ def _initialize_image_builder(init_config: state.BuilderInitConfig) -> None:
             timeout=10 * 60,
             env=os.environ,
         )  # nosec: B603
+    except subprocess.CalledProcessError as exc:
```

Comment:
Couldn't it be handled together with the SubprocessError?

Reply:
The reason for a separate exception is because CalledProcessError is more specific to the exit code where as Subprocess Error is more general (maybe even error creating a subprocess!). This allows the exception to carry over data (returncode, stdout and stderr) which is very helpful if an error occurs :)
Thank you for the review!